lesson_6/mosut_sorin_functions_homework.py

```diff
@@ -0,0 +1,201 @@
+# import pakcage for Excel files
```

Comment:
oare de ce ai utilizat pachetul de import Excel?

Reply:
Pt a transforma abrevierile tarilor in denumiri complete am folosit un document excel cu abrevieri.

Reply:
multumesc